server/controllers/payment.js

```diff
@@ -0,0 +1,190 @@
+const User = require("../models/User");
+const StripeServices = require("../services/StripePayment");
+const asyncHandler = require("express-async-handler");
+const {
+  stripePayment,
+  createCustomer,
+  getCustomer,
+  subscriptionOneTimePayment,
+  createIntent,
+  getPaymentMethods,
+} = StripeServices;
+
+// @route POST /:id/single-pay
+// @desc create single payment checkout session
+// @access Private
+exports.payment = asyncHandler(async (req, res, next) => {
```

Comment:
You may get rid of `next` function if not used

Reply:
👍🏻Done